NuGetPackageVerifier.json

```diff
@@ -16,6 +16,7 @@
             "tools/any/any/Microsoft.Build.Utilities.Core.dll": "Assembly built by another project but bundled in our nupkg.",
             "tools/any/any/MSBuild.dll": "Assembly built by another project but bundled in our nupkg.",
             "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/SdkResolvers/NuGet.MSBuildSdkResolver/NuGet.MSBuildSdkResolver.dll": "Assembly built by another project but bundled in our nupkg.",
```

Comment:
I'm surprised this is required. Why are we redistributing an SDK resolver?

Reply:
This is being brought in by `Microsoft.Build.Runtime/15.7.0-preview-000011-1378327`

Reply:
So, dotnet-aspnet-codegeneration has the entire MSBuild runtime inside it?

Reply:
Looks like it is packaging a lot of it, if not all of it. 
![image](https://user-images.githubusercontent.com/1521254/36456229-b63fb22c-1658-11e8-8e3e-b3d7f6af67d8.png)

Reply:
How big is dotnet-aspnet-codegeneration.nupkg?

It seems to me this dependency was already present. This just becomes more obvious because DotnetTool package requires flattening dependencies into your own package. What are you using the MSBuild API for? There may be simple ways to change this cli tool so you don't have to redistribute the entire .NET Core SDK inside the dotnet-aspnet-codegeneration package.

Reply:
Yeah its big right now (~3MB). Where as the old MS.VS.Web.CG.Tools package was only around 85 KB.

I will try to see if I can get rid of the MSBuild runtime dependency.